numba/unicode.py

```diff
@@ -570,6 +570,27 @@ def split_whitespace_impl(a, sep=None, maxsplit=-1):
         return split_whitespace_impl
 
 
+@overload_method(types.UnicodeType, 'ljust')
+def unicode_ljust(string, width, fillchar=' '):
```

Comment:
`fillchar` is weird, it is not actually a keyword argument:
```
$ python -c '"".ljust(1, fillchar="x")' 
Traceback (most recent call last):
  File "<string>", line 1, in <module>
TypeError: ljust() takes no keyword arguments
```
I don't think there is any simple fix for this.